DigitalLearningSolutions.Data/DataServices/SupervisorService.cs

```diff
@@ -992,25 +992,31 @@ public IEnumerable<CandidateAssessmentSupervisorVerificationSummary> GetCandidat
         {
             return connection.Query<CandidateAssessmentSupervisorVerificationSummary>(
                 @"SELECT	ca1.ID, 
-		                    AdminUsers.Forename, 
-		                    AdminUsers.Surname, 
-		                    AdminUsers.Email, 
-		                    COUNT(sas1.CompetencyID) AS VerifiedCount
-                    FROM   SelfAssessmentResultSupervisorVerifications AS sasrv
-                    INNER JOIN SelfAssessmentResults AS sar1 
-	                    ON sasrv.SelfAssessmentResultId = sar1.ID  AND sasrv.Superceded = 0
-                    INNER JOIN CandidateAssessmentSupervisors 
-	                    ON sasrv.CandidateAssessmentSupervisorID = CandidateAssessmentSupervisors.ID 
-                    INNER JOIN SupervisorDelegates sd
-	                    ON CandidateAssessmentSupervisors.SupervisorDelegateId = sd.ID 
-                    INNER JOIN AdminUsers 
-	                    ON sd.SupervisorAdminID = AdminUsers.AdminID 
-                    RIGHT OUTER JOIN SelfAssessmentStructure AS sas1 
-                    INNER JOIN CandidateAssessments AS ca1 
-	                    ON sas1.SelfAssessmentID = ca1.SelfAssessmentID 
-                    INNER JOIN CompetencyAssessmentQuestions AS caq1 
-	                    ON sas1.CompetencyID = caq1.CompetencyID 
-	                    ON sar1.SelfAssessmentID =sas1.SelfAssessmentID and sar1.CompetencyID=sas1.CompetencyID AND sar1.AssessmentQuestionID = caq1.AssessmentQuestionID AND sar1.DelegateUserID = ca1.DelegateUserID
+		                      AdminUsers.Forename, 
+                            AdminUsers.Surname, 
+                            AdminUsers.Email, 
+                            COUNT(sas1.CompetencyID) AS VerifiedCount, AdminUsers.Active AS AdminActive
+                    FROM   SelfAssessmentResultSupervisorVerifications AS sasrv
+                    INNER JOIN SelfAssessmentResults AS sar1 
+                        ON sasrv.SelfAssessmentResultId = sar1.ID  AND sasrv.Superceded = 0
+                    INNER JOIN CandidateAssessmentSupervisors 
+                        ON sasrv.CandidateAssessmentSupervisorID = CandidateAssessmentSupervisors.ID 
+                    INNER JOIN SupervisorDelegates sd
+                        ON CandidateAssessmentSupervisors.SupervisorDelegateId = sd.ID 
+                    INNER JOIN AdminUsers 

```

Comment:
AdminUsers is a SQL view which combines data from the new AdminAccounts and Users tables to recreate the deprecated and removed AdminUsers table.
We should remove references to the view here and query the AdminAccounts and Users table directly.
This would take the form of: 
`INNER JOIN AdminAccounts AS ac ON sd.SupervisorAdminID = ac.ID INNER JOIN Users AS u ON ac.UserID = u.ID`
Users will provide name and email address details, while the Active status will come from the AdminAccounts table.